types/index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for @nuxtjs/auth 4.8
+// Project: https://auth.nuxtjs.org
+// TypeScript Version: 3.1
+
```

Comment:
Can you add the new classes? There are `Token`, `RefreshToken`, `TokenStatus`, `RefreshController` and `RequestHandler`.

Reply:
do we really need RefreshController and RequestHandler? is the developer expected to interact with them or are they private to the module?

Reply:
We will add custom scheme support in v5. So developers can use these classes in their own schemes.